### DISCUSSION TITLE
Support for Quorum Queues

### DISCUSSION THREAD

**Discussion by martywaz** [rabbitmq/rabbitmq-delayed-message-exchange#285] [General]
Will the delayed Message Exchange plugin work if the back end Rabbit MQ environment is configured as Quorum Queues.

**Comment by lukebakken** [comment]
Have you tried it yourself?

**Reply by martywaz** [reply]
I've not had time or resources to build out a quorum yet.  But I have customers that do use Quorums and they wanted to know.

**Reply by michaelklishin** [reply]
@martywaz but our team has infinite resources to help out your customers who pay you and not us. Makes sense.

**Comment by michaelklishin** [comment]
This is an exchange type that delays publishing. It really does not care where the messages are routed at (re)publishing time.

As suggested above, you could have tried it very easily. Also note this plugin's non-replicated nature and limitations described in the README.
